modules/setting/history/breaking_changes.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package history
+
+import "code.gitea.io/gitea/modules/log"
+
+// This file is the only file that should be changed frequently in this package
+
+var currentGiteaVersion = getVersion("1.19")
```

Comment:
Wow, I don't think it's a good idea to ask maintainers to remember this.

Reply:
Could you please elaborate on how I get the information on what our current version is?
If I know how to do it, I can implement it.

Reply:
* Released binary: there is a var containing the build version (set by Makefile from `git describe`)
* Dev mode: `git describe`:

```
~/work/gitea$ git describe
v1.20.0-dev-52-g151d3e7832
```

Reply:
From #26094 
> The main problem of that PR is that I still haven't found out how I have to set LD_FLAGS so that the version will be updated automatically.

The version is currently set in `main` https://github.com/go-gitea/gitea/blob/6aa30af724b73c698b87c24fd5bb7d71b3c37abe/main.go#L28
https://github.com/go-gitea/gitea/blob/6aa30af724b73c698b87c24fd5bb7d71b3c37abe/Makefile#L108

To use it elsewhere you'll need to move it to a non-main package.

Reply:
The problem is especially that it isn't in the main package, and my attempts to convert it to `modules.setting.history.currentGiteaVersion` were all fruitless endeavors thus far.
That's why this PR has moved to the back of my TODO list as I haven't found out yet what to use.

Reply:
`-X code.gitea.io/gitea/modules/setting/history.currentGiteaVersion=$(GITEA_VERSION)` should do the trick, although I'd recommend exporting the variable so we can use the same one in both places.